mods/stairs/init.lua

```diff
@@ -46,12 +46,24 @@ end
 -- Node will be called stairs:stair_<subname>
 
 function stairs.register_stair(subname, recipeitem, groups, images, description, sounds)
+	local stair_images = {}
+	for i, image in ipairs(images) do
+		if type(image) == "string" then
+			stair_images[i] = {
+				name = image,
+				backface_culling = true,
+			}
+		elseif image.backface_culling == nil then -- override using any other value
```

Comment:
the comment is a little ambiguous, it's supposed to mean that "the code that sets it to true can be overriden (disabled) by using (setting) any other value (than `nil`) in the definition table"